jpasskit/src/main/java/de/brendamour/jpasskit/PKField.java

```diff
@@ -185,27 +185,52 @@ public boolean isValid() {
     public List<String> getValidationErrors() {
```

Comment:
It could be a good idea to call validation "business logic" and extract it from this data (model) layer. I mean that all validations could be moved to a separate utility class (or set of classes).

Reply:
I thought about that too. The only issue with that was that this essentially means you have to know about the validation utility in order to use it..

Reply:
Let's address this in another PR